java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/TestSuiteXmlRenderer.java

```diff
@@ -67,4 +71,12 @@ public void toXml(XMLStreamWriter xml, TestData suite, Collection<TestData> test
 
     xml.writeEndElement();
   }
+
+  private String getHostname() {
```

Comment:
The spec does callout that localhost can be used, I do not have a need to know the host, but figure I would attempt to set it in case anyone does need it